bulk_webincli.py

```diff
@@ -129,8 +132,52 @@ def get_args():
     if Path(args.webinCliPath).exists() is False:
         print("> ERROR: Cannot find the Webin CLI jar file. Please set the path to the Webin CLI jar file (--webinCliPath)")
         sys.exit()
+    if not fnmatch.fnmatch(args.webinCliPath, '*.jar'):
+        webinCli_file = webinCli_latest_download(args.webinCliPath)
+        args.webinCliPath = f'{args.webinCliPath}/{webinCli_file}'
     return args
 
+def webinCli_latest_download(webinCli_dir):
+    """
+        Checking and retrieving the latest Webin Cli jar file
+        :param: webinCli_dir: directory path for webin-cli
+        :return: Latest Webin Cli jar file name
+    """
+    print('checking if webin-cli is the latest release')
+    download_command = 'curl -s https://api.github.com/repos/enasequence/webin-cli/releases/latest |  grep "browser_download_url"  | head -1 | cut -d : -f 2,3 | tr -d \\"'
+    sp = subprocess.Popen(download_command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = sp.communicate()
+    webinCli_file_name = out.decode().split("/")
+    stdoutOrigin = sys.stdout
+    webinCli_list = glob.glob(f'{webinCli_dir}/*.jar')
```

Comment:
Add a comment to say what this is doing - I believe it's checking or any Webin-CLI jar files to see if they already exist, but explicitly mention in a comment.